plugin-server/src/utils/db/db.ts

```diff
@@ -642,10 +642,23 @@ export class DB {
         isIdentified: boolean,
         uuid: string,
         distinctIds?: string[],
-        version = 0
+        distinctIdVersions?: number[]
```

Comment:
This, and the stuff below, is a bit ugly. I think this function is maybe too flexible? We only pass in 1 or 2 Distinct IDs.

Anyway, I didn't want to add a larger refactor (much less all the existing tests...) right now. But I did want to add the ability to pass in the version *per Distinct ID* so we can avoid a lot of overrides...